### DISCUSSION TITLE
Is Lot# and vaccination site extractable from QR code?

### DISCUSSION THREAD

**Discussion by TheTravelGeek** [Digitaler-Impfnachweis/covpass-android#162] [Q&A]
I work at a Covid vaccination site in the US.  We occasionally see patients for booster shots who got their original series of vaccines in the EU and present their digital vaccine certs (sometimes paper, more often on phone).

With the CovPass Check app I can verify their vaccination status.  But it seems to provide limited information as it is targeted simply towards verification.

It seems (according to https://ec.europa.eu/health/sites/default/files/ehealth/docs/vaccination-proof_interoperability-guidelines_en.pdf) that the digital cert (QR) actually contains more information, targeting healthcare providers.

When we vaccinate people who can’t provide a paper vaccine card/passport (CDC card or WHO vaccine passport) to add information about the shot their receive from us, we issue a new US CDC card and copy the information about their previous vaccinations to make it as complete as possible.  

So my question is: is there a way to get 

- lot number
- vaccination site (clinic, location)

from the QR code, perhaps with another scan/verify app?

**Comment by vaubaehn** [comment]
@TheTravelGeek 
Hi, I'm not a developer, but I think, I can answer your question.
Unfortunately, it's not possible, as the Digital Covid Certificate (and its representation in form of a QR code) does not contain any information regarding the lot no., and only limited information regarding the vaccination site.
For DCCs issued in Germany, each DCC has a unique identifier that contains an identifier where the DCC had been issued. If people had been vaccinated on a vaccination site and received their DCC there, you will find the characters IZ followed by a zip code and a capital letter that would help to separate a number of vacc sites in the same district. But if they had been vaccinated and didn't receive their DCC on site, they would have gone to any pharmacy to get their DCC issued. You will be able to detect such certificates by the letter 'A' in the certificate, followed by some numbers that point to a larger area, and a number that represents the pharmacy in the area.
I don't know how other countries issuing DCCs handle it, but I'm sure there's rather a huge variation.
So, summed up, these information won't help you for your intended purpose.

**Comment by TheTravelGeek** [comment]
@vaubaehn - thanks for your reply.
